### ISSUE TITLE
Consistency Checks should be overridable on what result you want instead of defaulting to majority

### ISSUE THREAD

**Issue by michaelst** [rrrene/credo#871]
### What were you trying to do?

We were adding the unused variables check to our enforced checks but it said everything should be an underscore when we really wanted all variables to be named.

### Expected outcome

Have an option to override the default of majority.

**Comment by rrrene** [comment]
Thanks for reporting this 😀  It is now fixed on `master`.

You can try this by setting the Credo dep to

```elixir
{:credo, github: "rrrene/credo"}
```

and setting the check's config to

```elixir
{Credo.Check.Consistency.UnusedVariableNames, [force: :meaningful]},
```

Please report back if your issue is solved! Then we could adapt this to all consistency checks 👍

**Comment by michaelst** [comment]
I tested this, but when I force anonymous it doesn't give me any warnings, I don't have have to test `force: meaningful` as I already have everything set to be named

**Comment by rrrene** [comment]
You are right. 🤦  There was a bug I had to fix ...

Could you test again?

**Comment by michaelst** [comment]
I'm still not getting any warnings with anonymous

**Comment by michaelst** [comment]
Yep, it is working now

**Comment by rrrene** [comment]
This is part of v1.6.1 💚
